addons/website_event_crm_questions/models/event_registration.py

```diff
@@ -15,10 +17,10 @@ def _get_lead_description_registration(self, line_suffix=''):
 
         answer_descriptions = []
         for answer in self.registration_answer_ids:
-            answer_value = answer.value_answer_id.name if answer.question_type == "simple_choice" else answer.value_text_box
-            answer_value = "<br/>".join(["    %s" % line for line in answer_value.split('\n')])
-            answer_descriptions.append("  - %s<br/>%s" % (answer.question_id.title, answer_value))
-        return "%s%s<br/>%s" % (reg_description, _("Questions"), '<br/>'.join(answer_descriptions))
+            answer_value = answer.value_answer_id.name if answer.question_type == "simple_choice" else html_sanitize(answer.value_text_box)
```

Comment:
Not sure to understand ... shouldn't we use markup and exclude value_text_box from it to be sure it will be escaped instead ? Why sanitizing ?

Reply:
Indeed, I mixed 2 fixes. Updated :+1: